ng2-components/ng2-activiti-tasklist/src/components/activiti-tasklist.component.ts

```diff
@@ -166,7 +166,7 @@ export class ActivitiTaskList implements OnChanges, AfterContentInit {
             changed = true;
         } else if (assignment && assignment.currentValue) {
```

Comment:
this piece of code need to be refactored all this if are not a good paradigm, can you please change it?